### PR TITLE
Update libunwind to 11.0.0

### LINKED ISSUE #13094: libunwind is out of sync with other libraries we pull from llvm
According to system/lib/libunwind/readme.txt we are on version 8.0.0 where as compiler-rt, libc++, and libc++abi are all on 10.0.0.

### PR DESCRIPTION
**Pull request by aheejin**
Fixes #13094.

### CODE REVIEW

**Comment by aheejin** [comment]
This currently contains contents of #13097 (because Github does not support stacked PRs..). This will be rebased onto it after it lands.

File: system/lib/libunwind/test/lit.cfg.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# All the Lit configuration is handled in the site configs -- this file is only
```
Comment (sbc100) [inline system/lib/libunwind/test/lit.cfg.py:1]: Maybe remove the entire "test" directory?

**Review by sbc100 — APPROVED** [review]
Nice.  Thanks!

File: system/lib/libunwind/test/lit.cfg.py (hunk 1 above)
Comment (aheejin) [reply system/lib/libunwind/test/lit.cfg.py]: Done

**Pull request merged**